openpype/hosts/maya/plugins/load/load_arnold_standin.py

```diff
@@ -99,7 +107,9 @@ def load(self, context, name, namespace, options):
             sequence = is_sequence(os.listdir(os.path.dirname(repre_path)))
             cmds.setAttr(standin_shape + ".useFrameExtension", sequence)
 
-            fps = float(version["data"].get("fps"))or get_current_session_fps()
+            fps = (
+                float(version["data"].get("fps")) or get_current_session_fps()
+            )
```

Comment:
The `.get("fps")` logic kind of enforces the `fps` key to exist because `float(None)` fails anyway.

```
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: float() argument must be a string or a number, not 'NoneType'
```
As such, either this may fail in cases where we don't want it to - or the `.get` logic is redundant.

Reply:
fixed